appveyor.yml

```diff
@@ -32,13 +32,8 @@ environment:
   # See: http://www.appveyor.com/docs/installed-software#python
   matrix:
     - PYTHON: C:\Python27-x64
-      TOXENV: py27-coverage-windows,codecov-publish
+      TOXENV: py27-coverage-windows,py27-nomodulescov-windows,codecov-publish
```

Comment:
Also, maybe it better to have it as a separate job (builder) and not run it together with `py27-coverage-windows`

when we run 2 different tests set in the same job, we end up with logs which are hard to read... and this is one of the reason why we moved away from tox-travis